rest/admin_api_test.go

```diff
@@ -358,7 +358,7 @@ function(doc, oldDoc) {
 
 		since := ""
 
-		maxTries := 10
+		maxTries := 30
```

Comment:
There's already a 2 second wait on each try, isn't there?  (timeout=2000)  It sounds correct to fail the test if the results don't arrive in 20s, even under race.

Reply:
I reverted this part of the change.

However, I'm not able to repro this locally by running it in a loop, so I think the most pragmatic approach forward is to merge this PR as-is and use the additional logging information to debug it if/when it happens again.